### ISSUE TITLE
Extracted features does not match the split in COCO dataset

### ISSUE THREAD

**Issue by sean-zhuh** [peteanderson80/bottom-up-attention#92]
Hi, I've downloaded the pretained features for train/val using the provided links (36 - 100 features per image), however, there are four tsv files in it, namely,

1) karpathy_train_resnet101_faster_rcnn_genome.tsv.0
2) karpathy_train_resnet101_faster_rcnn_genome.tsv.1
3) karpathy_val_resnet101_faster_rcnn_genome.tsv
4) karpathy_test_resnet101_faster_rcnn_genome.tsv

each of the four file contains 56643,56644,5000,5000 unique image_id, however, in vqa2.0, training dataset has 82783 image, validation set contains 40,504 images, how can I know which image_id is for training set, which is for validation set?

**Comment by nihirv** [comment]
Struggling with this too. I've loaded in the trainval files and the test files but am still struggling to find images which were in the original 2014 validation set

**Comment by haixiongli** [comment]
Hello, I encountered the same problem. I don't know how the csv file here corresponds to the image id in the coco dataset. Have you solved this problem?
